### ISSUE TITLE
Can't display docker image tag list once default scanner crushed and image has scan report

### ISSUE THREAD

**Issue by MaGaoju** [goharbor/harbor#10504]
**Expected behavior and actual behavior:**
Expected behavior：image should displayed in the UI . 
![image](https://user-images.githubusercontent.com/13995442/72504561-01c35c00-3879-11ea-95f0-f455aa73c60d.png)

actual behavior: 
empty content display with error message on top 
![image](https://user-images.githubusercontent.com/13995442/72504456-cd4fa000-3878-11ea-8fdf-065d327930a2.png)

Core error log:

2020/01/16 07:38:17 Failed to get scan report summary with error: scan controller: get report: scanner controller: ping: v1 client: get metadata: invalid character '<' looking for beginning of value: general response handler: unexpected status code: 500, expected: 200
2020-01-16T07:38:17Z [ERROR] [/pkg/scan/api/scanner/base_controller.go:229]: api controller: get project scanner: scanner controller: ping: v1 client: get metadata: invalid character '<' looking for beginning of value: general response handler: unexpected status code: 500, expected: 200
2020/01/16 07:38:18 Failed to get scan report summary with error: scan controller: get report: scanner controller: ping: v1 client: get metadata: invalid character '<' looking for beginning of value: general response handler: unexpected status code: 500, expected: 200
2020-01-16T07:38:18Z [ERROR] [/pkg/scan/api/scanner/base_controller.go:229]: api controller: get project scanner: scanner controller: ping: v1 client: get metadata: invalid character '<' looking for beginning of value: general response handler: unexpected status code: 500, expected: 200
2020-01-16T07:38:18Z [ERROR] [/pkg/scan/api/scanner/base_controller.go:229]: api controller: get project scanner: scanner controller: ping: v1 client: get metadata: invalid character '<' looking for beginning of value: general response handler: unexpected status code: 500, expected: 200
2020/01/16 07:38:18 Failed to get scan report summary with error: scan controller: get report: scanner controller: ping: v1 client: get metadata: invalid character '<' looking for beginning of value: general response handler: unexpected status code: 500, expected: 200
2020-01-16T07:38:18Z [ERROR] [/pkg/scan/api/scanner/base_controller.go:229]: api controller: get project scanner: scanner controller: ping: v1 client: get metadata: invalid character '<' looking for beginning of value: general response handler: unexpected status code: 500, expected: 200
2020/01/16 07:38:18 Failed to get scan report summary with error: scan controller: get report: scanner controller: ping: v1 client: get metadata: invalid character '<' looking for beginning of value: general response handler: unexpected status code: 500, expected: 200
2020-01-16T07:38:20Z [ERROR] [/pkg/scan/api/scanner/base_controller.go:229]: api controller: get project scanner: scanner controller: ping: v1 client: get metadata: invalid character '<' looking for beginning of value: general response handler: unexpected status code: 500, expected: 200
2020/01/16 07:38:20 Failed to get scan report summary with error: scan controller

**Steps to reproduce the problem:**
scanner adapter can't connect to scanner, and this adapter has been set as default scanner in harbor. visit harbor project, check the image which already has report generated by this scanner. 

**Versions:**

- harbor version: 1.10.0
- harbor helm version: 1.3.0 
![image](https://user-images.githubusercontent.com/13995442/72504953-d8ef9680-3879-11ea-9675-f252346bcbd5.png)

**Comment by cafeliker** [comment]
Can Harbor remove such strong dependency on the scanner? Any issue on scanner side should not impact the Harbor user experience.

An alternative approach is switch the scanner to another available one, e.g. switch from Anchore to the Clair.

Thanks
ye

**Comment by steven-zou** [comment]
@MaGaoju @cafeliker 

Please recheck if your containers/pods of the harbor registry are working well. 

The scan errors printed in the log are just for debugging purpose, they will not cause the request to fail.

```go
        sum, err := scan.DefaultController.GetSummary(artifact, []string{v1.MimeTypeNativeReport})
	if err != nil {
		logger.Errorf("Failed to get scan report summary with error: %s", err)
	}
```

**Comment by MaGaoju** [comment]
@steven-zou  registry works, just web UI has above error message. I have export the core and registry logs during the same period which has this error. hope this helps. and let me know if you need more information. 


[log-events-core-141446.txt](https://github.com/goharbor/harbor/files/4266076/log-events-core-141446.txt)
[log-events-registry-141446.txt](https://github.com/goharbor/harbor/files/4266077/log-events-registry-141446.txt)

**Comment by steven-zou** [comment]
@heww 

Please double-check if the issue mentioned here is still there. I think it should be fixed.

**Comment by cafeliker** [comment]
@steven-zou we met the exact same issue in production today. and when you were saying "it should be fixed" - means it's fixed in Harbor 2.0?

**Comment by heww** [comment]
> @heww
> 
> Please double-check if the issue mentioned here is still there. I think it should be fixed.

I had check the source code that we will still ping the scanner to get the report in v2.0 so this problem still exist in v2.0
